diff --git a/out/package.js b/out/package.js
index d9cdfc6bd43dc067846e327a5fc200bb6be8b002..9b57feb47281078cc3feef69902aac320c994ba0 100644
--- a/out/package.js
+++ b/out/package.js
@@ -1211,7 +1211,7 @@ const defaultIgnore = [
 ];
 async function collectAllFiles(cwd, dependencies, dependencyEntryPoints, followSymlinks = true) {
     const deps = await (0, npm_1.getDependencies)(cwd, dependencies, dependencyEntryPoints);
-    const promises = deps.map(dep => (0, glob_1.glob)('**', { cwd: dep, nodir: true, follow: followSymlinks, dot: true, ignore: 'node_modules/**' }).then(files => files.map(f => path.relative(cwd, path.join(dep, f))).map(f => f.replace(/\\/g, '/'))));
+    const promises = deps.map(dep => (0, glob_1.glob)('**', { cwd: dep, nodir: true, follow: followSymlinks, dot: true }).then(files => files.map(f => path.relative(cwd, path.join(dep, f))).map(f => f.replace(/\\/g, '/'))));
     return Promise.all(promises).then(util.flatten);
 }
 function getDependenciesOption(options) {
